### PR TITLE
update python script to support Python3

### PR DESCRIPTION
**Pull request by lifegpc**
I test it with Python 3.7.4. Works well.

### CODE REVIEW

**Comment by clsid2** [comment]
I assume this is not backwards compatible with v2?
In that case the compilation instructions document also needs to be updated.

**Comment by lifegpc** [comment]
yea.

**Comment by lifegpc** [comment]
done.

File: build.user_example.bat [hunk 1]
```diff
@@ -7,8 +7,8 @@ SET "MSYSTEM=MINGW32"
 SET "MSYS2_PATH_TYPE=inherit"
 REM [Optional] Specify GIT location if it is not already set in %PATH%
 SET "MPCHC_GIT=C:\Program Files\Git"
-REM [Optional] If you plan to modify the translations, install Python 2.7 and set the variable to its path
-SET "MPCHC_PYTHON=C:\Python27"
+REM [Optional] If you plan to modify the translations, install Python 3.7 and set the variable to its path
+SET "MPCHC_PYTHON=C:\Program Files\Python37"
```
Comment (ghost) [inline build.user_example.bat]: It's better to use Python 3.9.1 (currently latest version) or vaguely use Python 3.x or later

File: docs/Compilation.md [hunk 2]
```diff
@@ -18,13 +18,13 @@
     When choosing which features to install you only need to select "Windows Software Development Kit".
 
 
-## Part B: Install Python 2.7 (optional)
+## Part B: Install Python 3 (optional)
 
 This is required for building the translation DLL files.
 
-1. Install Python version 2.7.18 from <https://www.python.org/downloads/release/python-2718/>
+1. Install Python version 3.7.9 from <https://www.python.org/downloads/release/python-379/> (You can use Python 3.6 or later version)
```
Comment (ghost) [inline docs/Compilation.md]: It's better to use the latest version 3.9.1 in the document.

**Review by ghost — CHANGES_REQUESTED** [review]
It's better to use the currently latest version (3.9.1) of Python in the document.

**Comment by lifegpc** [comment]
Maybe I can change it to 3.8. 3.9 can not used on win7.

**Comment by clsid2** [comment]
3.8.7 works in Win7

**Pull request merged**